src/main/kotlin/com/theworkshopvlc/chupapuntes/categories/CategoriesController.kt

```diff
@@ -20,7 +20,7 @@ class CategoriesController(private val getAllCategories: GetAllCategories,
     return getAllCategories.execute()
   }
 
-  @GetMapping("/getByTitle")
+  @GetMapping
```

Comment:
I don't know if this will work because there is a method already hooked for the root request of the controller which is listing all the categories.

Reply:
"/index" is "/categories"? I thought it worked like "/categories/index"

Reply:
This isn't like Ruby on Rails. If you only annotate a method with `@GetMapping` and no specific route that method will listen for requests to the root path. In this case, `CategoriesController` is annotated with `@RequestMapping("/categories")` so this method will listen to GET requests to "/categories". There is also another method which is "/index" so it will listen to "/categories/index". But that method lists all the categories in database, that kind of requests usually go in the root path of an entity in REST APIs. We should change both methods.

Anyway is late and I'm not in my best for reviewing this now. Maybe tomorrow I come up with some solution.